### PR TITLE
Responsiveness

### LINKED ISSUE #1: Make it responsive
There are 2 html pages 
```
index.html
about.html
```
Main css file in static folder contains dynamic star animations
```
style.css
```

### PR DESCRIPTION
**Pull request by ghost**
Fix #1

### CODE REVIEW

**Review by akhileshthite — CHANGES_REQUESTED** [review]
Issue 1:- The nev-bar is not resizing/ responsive on the index.html page

Issue 2:- Colour of the star-background is different (grey) than the original, keep it original.

File: static/style.css [hunk 1]
```diff
@@ -1,36 +1,194 @@
...
+}
+
+.footer {
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  bottom: 0;
+  padding: 10px 10px;
+  width: 100vw;
+  background-color: rgba(0, 0, 0, 0.75);
```
Comment (akhileshthite) [inline static/style.css:134]: Appearing 2 different misplaced backgrounds on the index.html

File: about.html [hunk 2]
```diff
@@ -1,104 +1,42 @@
...
+  <link rel="stylesheet" href="./static/style.css" />
 </head>
+
 <body>
-<div class="topnav">
-    <a href="index.html">Home</a>
-    <a href="https://akhileshthite.github.io/Portfolio-Projects/">Projects</a>
-    <a class="active" href="/about">About</a>
+  <div class="navbar">
+    <a data-active="false" href="/home">Home</a>
```
[the remark this answers is not in the document]
Comment (ghost) [reply about.html]: Fixed It

File: static/style.css [hunk 3]
```diff
@@ -1,36 +1,194 @@
...
+.navbar a[data-active="true"] {
+  background-color: #4caf50;
+  color: white;
+}
+
+.h1 {
+  background-color: #ffffff00;
+  color: white;
+  font: 60px arial;
+  margin-bottom: 20px;
```
[the remark this answers is not in the document]
Comment (ghost) [reply static/style.css]: Fixed it: Changed to 10px

File: static/style.css (hunk 1 above)
Comment (ghost) [reply static/style.css:134]: Fixed This

File: static/style.css [hunk 4]
```diff
@@ -1,36 +1,194 @@
...
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  background-color: white;
+  text-align: center;
+  transition: 0.5s;
+  box-shadow: 0px 7px 5px rgba(0, 0, 0, 0.5);
+}
+
+.info {
```
[the remark this answers is not in the document]
Comment (ghost) [reply static/style.css]: Fixed This aswell

**Comment by ghost** [comment]
Note: That everything is responsive till `310px: IPhone 5` the smallest screen size.

**Review by akhileshthite — APPROVED** [review]
Great work!

**Pull request merged**